R/prep_and_calc_loan_book.R

```diff
@@ -116,7 +119,7 @@ run_prep_calculation_loans <- function(credit_type = "outstanding") {
         ald_timestamp = "c"
       )
     )
-  } else if (start_year == 2021) {
+  } else if (start_year == 2020) {
```

Comment:
I do not entirely understand what is going on but:

- this will change the ald data we use. This means our pacta_results and sector_exposures as used in the main workflow will change also, are any structural changes possible?
- is it realistic we have to support 2019 in general?
- did you verify all other aspects that depend on start year where changed accordinglt?

Reply:
I am ware this will change it, which again is due to my previous misconception that years are handled equally i P4I and P4B. I is correct now and will indeed change the values in the LOANS test case. It is a valid point that we may not have to support 2019 at all. I don't think anyone will be using it, but will check in with George

Reply:
only 2020 left now